src/dhwcalc.cpp

```diff
@@ -3400,35 +3393,35 @@ RC HPWHLINK::hw_DeriveVolFromVolRunning(		// calc required volume from running v
 // Does not actually set volume
 // returns RCOK iff success
 {
-	RC rc = RCOK;
-
 	// retrieve tank volume fractions
 	//   apply insurance (crash-proof) limits
 	double aquaFract;	// fraction of volume below aquastat
 	double useableFract;	// fraction of volume that is useable
-	if (hw_pHPWH->getSizingFractions(aquaFract, useableFract) != 0)
-	{	rc |= RCBAD;
-		aquaFract = .4f;		// plausible values
-		useableFract = .9f;
-	}
-	useableFract = bracket(.6, useableFract, 1.);
-	double unuseableFract = 1. - useableFract;
-	aquaFract = bracket(unuseableFract + .1, aquaFract, .75);
 
-	// total volume req'd based on aquastat position
-	//    Running vol is vol above aquastat
-	float totVolRun = float(volRunning / (1. - aquaFract));
+    if (hw_pHPWH->getSizingFractions(aquaFract, useableFract) != 0) {
```

Comment:
It looks like this function always returns zero. It should be modified to either return the two fractions or be a void function.